### PR TITLE
All Products Types View Model

### PR DESCRIPTION
**Pull request by HubDub**
## Pull Request Description

Include the applicable information from the list below here 
The AllProductTypes view model should pass all of the product types to the product types view.
This class has a property of AllProductTypes and a constructor method that passes the BangazonWeb Context in as an argument
## Steps to Test

Include the applicable information from the list below here
1. Examine code in PR for errors.
## Link to Feature Ticket
#79

### CODE REVIEW

**Review by Jgregnier — COMMENTED** [review]
Other than the method name being off it looks good to me

**Review by Jgregnier — COMMENTED** [review]
Class name is still "Products" should be AllProductTypesViewModel. Other than that looks good to me

File: ViewModels/AllProductTypesViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using BangazonWeb.Models;
+using BangazonWeb.Data;
+
+namespace BangazonWeb.ViewModels
+{
+    //Class Name: Products
```
Comment (delaineackerman) [inline ViewModels/AllProductTypesViewModel.cs]: AllProductTypesViewModel should be the class name

File: ViewModels/AllProductTypesViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    //Author: Debbie Bourne
+    //Purpose of the class: The purpose of this class is to manage the property and method that will produce the data and functionality needed for the view of all of the product types
```
Comment (delaineackerman) [inline ViewModels/AllProductTypesViewModel.cs:9]: Maybe say: the property - ProductTypes - that will be set in the controller to produce the data...

**Comment by thrashMoneyVillainare** [comment]
LookinGuud

**Review by JamsDuke — COMMENTED** [review]
Code looks good!

**Review by delaineackerman — COMMENTED** [review]
LGTM

**Review by csmalley89 — APPROVED** [review]
👍

**Pull request merged**